app/views/performance_reports/show.html.slim

```diff
@@ -37,7 +37,20 @@ table.table
       td
     tr.active
       td= t('.volunteers.active_total')
-      td= @report_content['global']['volunteers']['active']
+      td= @report_content['global']['clients']['active']
```

Comment:
Shouldn't then the above line change as well? ` td= t('.volunteers.active_total')`?

Reply:
right!